lib/CFRMessageProvider.jsm

```diff
@@ -284,7 +289,7 @@ const CFR_MESSAGES = [
           label: {string_id: "cfr-doorhanger-extension-manage-settings-button"},
           action: {
             type: "OPEN_PREFERENCES_PAGE",
-            data: {category: "general-cfr", origin: "CFR"},
```

Comment:
you probably want to check with @ncloudioj to make sure this change won't affect any of our data / is OK

Reply:
I've only changed the `category` field used to target which section and option in `about:preferences` to open to and highlight. The `origin` field used for telemetry is the same.

https://searchfox.org/mozilla-central/rev/00f3836a87b844b5e4bc82f698c559b9966e4be2/browser/base/content/utilityOverlay.js#856

Reply:
This change won't affect the telemetry.

@piatra This patch needs a data review, because it also added two events (`MANAGE` and 'PIN_CURRENT_TAB') to the telemetry.